k2/csrc/ragged.h

```diff
@@ -594,6 +608,11 @@ RaggedShape TrivialShape(ContextPtr &c, int32_t num_elems);
 RaggedShape RaggedShapeFromTotSizes(ContextPtr &c, int32_t num_axes,
                                     int32_t *tot_sizes);
 
+inline RaggedShape RaggedShapeFromTotSizes(ContextPtr &c, std::vector<int32_t> &tot_sizes) {
+  return RaggedShapeFromTotSizes(c, tot_sizes.size(), &(tot_sizes[0]));
```

Comment:
seems there is a static_cast from `size_t` to `int32_t` for `tot_sizes.size()`, not sure if there's compilation warning.